scripts/engine/world.py

```diff
@@ -82,20 +83,21 @@ def create_god(god_name: str) -> EntityID:
     return entity
 
 
-def create_actor(name: str, description: str, tile_pos: Tuple[int, int], trait_names: List[str],
+def create_actor(name: str, description: str, occupying_tiles: List[Tuple[int, int]], trait_names: List[str],
         is_player: bool = False) -> EntityID:
     """
     Create an entity with all of the components to be an actor. Returns entity ID.
     """
     components: List[Component] = []
+    tile_pos = occupying_tiles[0]
     x, y = tile_pos
 
     # actor components
     if is_player:
         components.append(IsPlayer())
     components.append(IsActor())
+    components.append(Position(*occupying_tiles))
```

Comment:
Do we always need to unpack to `Position` now?

Reply:
I can change it to a list if you feel more comfortable with that.

Reply:
I think I just need to get used to it, no need to change it.

Reply:
In this case it needed to be unpacker because it was a list but when you are giving it directly the coordinates there is no need to unpack e.g. It can be
```python
components.append(Position((0,0), (0, 1), (1,1), (1,2)))
```
There is an example in the new test